andela/coding-geeks-ah-backend
# #167484581 integrated authors haven with coveralls coverage service

#### What does this PR do?
It integrates **Authors Haven** with **Coveralls** coverage service.

#### Description of Task to be done?
1. Setting up required dependencies/packages.
2. Adding coverage script.
3. Adding coverage badge in the readme.
3. Creating coveralls YAML file for coveralls integration configuration.

#### Screenshots
<img width="557" alt="Screen Shot 2019-07-31 at 14 57 35" src="https://user-images.githubusercontent.com/34879554/62214452-6101e280-b3a5-11e9-9561-e32513746019.png">


#### How should this be manually tested?
1. Clone this branch.
2. Do **npm install**.
3. View and check if you see the coverage badge with unknown status and also check in the readme if it links to the repo: coding-geeks-ah-backend
